server/controllers/phrases.ts

```diff
@@ -0,0 +1,117 @@
+import Nouns from '../models/Nouns'
+import Adjectives from '../models/Adjectives'
+import nouns from '../data/nouns'
+import adjectives from '../data/adjectives'
+
+import express, { Request, Response } from 'express'
+
+const phraseRouter = express.Router()
+phraseRouter.use(express.json())
+
+phraseRouter.get('/noun', async (request: Request, response: Response) => {
+  //hardcoded total number of nouns
+  var random = Math.floor(Math.random() * 2283)
```

Comment:
it's usually a good idea to store hard-coded values in named variables in a common place, either at the top of the file or in a file specifically for constants so they're easy to find and manage